source/_integrations/androidtv_remote.markdown

```diff
@@ -32,21 +37,24 @@ The entity has the `current_activity` attribute that shows the current foregroun
 
 ## Services
 
-You can use the `remote.turn_off`, `remote.turn_on`, `remote.toggle`,  and `remote.send_command` services from the [remote](/integrations/remote/) platform.
+You can use the `remote.turn_off`, `remote.turn_on`, `remote.toggle`,  and `remote.send_command` services
+from the [remote](/integrations/remote/) platform.
 
-For a list of the most common commands that you can send to the Android TV via `remote.send_command`, see: [TvKeys](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt).
-For a full list, see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90).
+For a list of the most common commands that you can send to the Android TV via `remote.send_command`,
+see: [TvKeys](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt).
```

Comment:
Same comment as the previous occurance, we shouldn't be linking people into source code/repositories. The documentation should cover things like this.